keys/keys_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Coinbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package keys
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGenerateKeypairSecp256k1(t *testing.T) {
+	curve := types.Secp256k1
+	keypair, err := GenerateKeypair(curve)
+
+	assert.NoError(t, err)
+	assert.Equal(t, keypair.PublicKey.CurveType, curve)
+	assert.Equal(t, keypair.PrivateKey.CurveType, curve)
+
+	privKeyJSON, _ := json.Marshal(keypair.PrivateKey)
+	var privKey PrivateKey
+	err = json.Unmarshal(privKeyJSON, &privKey)
+	assert.NoError(t, err)
+	assert.Equal(t, privKey.Bytes, keypair.PrivateKey.Bytes)
```

Comment:
nit: expected should go first:
```suggestion
	assert.Equal(t, keypair.PrivateKey.Bytes, privKey.Bytes)
```